src/components/products/Products.component.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from 'react';
+
+import { FiltersBar } from '@/components/filtersBar/FiltersBar.component.tsx';
+import { Loading } from '@/components/loading/Loading.component.tsx';
+import { Pagination } from '@/components/pagination/Pagination.component.tsx';
+import { ProductList } from '@/components/productList/ProductList.component.tsx';
+
+export const Products = () => {
+    const [data, setData] = useState([]);
+    const [query, setQuery] = useState('');
+    const [searchInput, setSearchInput] = useState('');
+    const [totalPage, setTotalPage] = useState(1);
+    const [currentPage, setCurrentPage] = useState(1);
+    const countProductsOnPage = 8;
+    const [isLoading, setLoading] = useState(true);
+    const [filter, setFilter] = useState(0);
+    // eslint-disable-next-line sonarjs/no-duplicate-string
+    const [sortOption, setSortOption] = useState('Price (High - Low)');
+
+    useEffect(() => {
+        const getAllProducts = async () => {
+            try {
+                const response = await fetch(`https://ma-backend-api.mocintra.com/api/v1/products`);
```

Comment:
why do you use this request two times? you can use this one time without params and with params. This is extra code